dev/create-release/spark-rm/Dockerfile

```diff
@@ -34,7 +34,7 @@ ENV DEBCONF_NONINTERACTIVE_SEEN true
 ARG APT_INSTALL="apt-get install --no-install-recommends -y"
 
 ARG BASE_PIP_PKGS="setuptools wheel virtualenv"
-ARG PIP_PKGS="pyopenssl pypandoc numpy pygments sphinx"
+ARG PIP_PKGS="pyopenssl pypandoc numpy sphinx"
```

Comment:
Sphinx still needs `pygments` but it's in sphinx's dependency. I don't think we should explicitly list since Spark doesn't use it (but sphinx uses it)

Reply:
cc @vanzin too